docs/examples.md

```diff
@@ -164,6 +164,37 @@ jobs:
           comment-summary-in-pr: always
 ```
 
+## Getting the results of the action in a later step
+
+Using the `comment-content` output you can get the results of the action in a workflow step.
```

Comment:
This precise workflow is untested.

The workflow I used is here:
https://github.com/check-spelling-sandbox/dependency-review-action-0/blob/657ea37667b5afdb8c94c4197272ba19adbfa0af/.github/workflows/dependency-review.yml

I _believe_ they're functionally equivalent, but someone should really carefully review (or better, test).